casbin/model/policy.py

```diff
@@ -154,7 +154,8 @@ def add_policies(self, sec, ptype, rules):
                 return False
 
         for rule in rules:
-            self[sec][ptype].policy.append(rule)
+            if not self.add_policy(sec, ptype, rule):
```

Comment:
You should fix all similar issues, including but not limited to: update_policies, remove_policies, remove_filtered_policy